### PR TITLE
WIP: Add a new game button at the end

### LINKED ISSUE #5403: Add a link to start a new game in the end game page
Something like this:

image

What do you think?

Here is an example of the code I used for this demo. If it works for the project, I can open PR.

HTML:
```HTML
<div class="game_end_go_home">
	<div class="whatever_suited_name">
		<a href="/new-game">
			<button class="btn btn-lg btn-rounded">
				<span data-test="icon" class="icon icon-back"></span>
			</button>
			<span>Start a new game</span>
		</a>
		<a href="/">
			<button class="btn btn-lg btn-rounded">
				<span data-test="icon" class="icon icon-back"></span>
			</button>
			<span>Go to main page</span>
		</a>
	</div>
</div>
```

CSS:
```CSS
.whatever_suited_name {
    display: flex;
    flex-direction: column;
}

.game_end_go_home {
    font-size: 30px;
    margin: 40px 0;
    display: flex;
    justify-content: center;

    button {
        margin-right: 10px;
    }
}
```

We could even add a restart button later on.

### PR DESCRIPTION
**Pull request by aloisdg**
Demo:

screenshot

Fix #5403

todo:

- [x] rename `whatever_suited_name` (any idea?)
- [x] handle translation (what should I do?)

### CODE REVIEW

File: src/client/components/GameEnd.vue [hunk 1]
```diff
@@ -36,10 +36,17 @@
               </div>
           </div>
           <div class="game_end_go_home">
+            <div class="whatever_suited_name">
```
Comment (kberg) [inline src/client/components/GameEnd.vue:39]: Here are three:`navigation`, `back`, `actions`. Pick one, or pick anything else you like.

**Review by kberg — APPROVED** [review]

File: src/client/components/GameEnd.vue (hunk 1 above)
Comment (aloisdg) [reply src/client/components/GameEnd.vue:39]: Well I think we could replace

```html
          <div class="game_end_go_home">
            <div class="whatever_suited_name">
```
with

```html
            <div class="game_end_navigation_wrapper">
              <div class="game_end_navigation">
```

Comment (kberg) [reply src/client/components/GameEnd.vue:39]: Whatever you prefer, this is a nice addition.

**Review by kberg — APPROVED** [review]
Nicely done. Thanks!

**Pull request merged**